### DISCUSSION TITLE
The remote session for the macOS system is displaying a blank screen.

### DISCUSSION THREAD

**Discussion by amoljagdalepucsd** [Ylianst/MeshCentral#5590] [Q&A]
Hi community , 
 When attempting to initiate a remote session on macOS system after logout or when there's no active user logged in. Instead of a login interface or session initiation, I consistently encounter a blank screen. This differs from my experience on Windows systems, where a login interface or session initialization occurs in similar scenarios.

Is anyone else experiencing this issue on macOS? And if so, have you found any solutions or workarounds to address this blank screen problem during remote sessions

**Comment by si458** [comment]
the is already issues asking about this (will attach more when i find them) - #3213 #1459
i believe the is a BUG where if the mac is on a LOGIN screen, then its not possible to control/access it
so you have to have the mac autologin to avoid this issue

**Reply by amoljagdalepucsd** [reply]
Thanks @si458  for the reply.
I am just going through agent side files related to macOS. will check is there any way to initiate session in this scenario.

**Reply by NiceGuyIT** [reply]
> will check is there any way to initiate session in this scenario.

Asking for permissions was done in Ylianst/MeshAgent#181 but wasn't implemented correctly and the fix is Ylianst/MeshAgent#215 which is waiting to be merged.

**Reply by amoljagdalepucsd** [reply]
Thank you @NiceGuyIT  for the reply.
But I am facing this blank screen only When attempting to initiate a remote session on macOS system after logout or when there's no active user logged in.
Will this issue get resolved by #215 PR.
@si458

**Comment by silversword411** [comment]
https://ylianst.github.io/MeshCentral/meshcentral/faq/#mac-clients

**Reply by amoljagdalepucsd** [reply]
Hi Community,

While exploring, I came across a feature called web-vnc. Clicking on this feature opens a new window on a non-VNC page, prompting for login credentials. Once the credentials are entered, it grants access to control a system, even if it's at the login screen without any logged-in user.

I'm curious if anyone has more information about this web-vnc feature. Specifically, I'm wondering if it's possible to integrate this functionality directly under the remote desktop option within MeshCentral for Mac machines, rather than providing it as a separate option.

If anyone has insights or experiences with integrating web-vnc under the remote desktop feature.
